server/app/controllers/NewTask.scala

```diff
@@ -20,6 +23,7 @@ class NewTask @Inject()(executor: DynamicSuiteExecutor, dao: Dao, val messagesAp
                        (implicit s: Scheduler) extends Controller with I18nSupport {
   val addTaskForm = Form {
     mapping(
+      taskName -> nonEmptyText,
```

Comment:
Even Jenkins Job has name. Do you think our users will struggle so much to put the names? :-)